### PR TITLE
Jakarta namespace transition

### PR DESCRIPTION
**Pull request by jGauravGupta**
Signed-off-by: Gaurav Gupta <gaurav.gupta@payara.fish>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,10 +34,28 @@
...
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish</groupId>
+                <artifactId>jakarta.json</artifactId>
+                <version>${jakarta.json.version}</version>
+                <classifier>module</classifier>
+                <scope>compile</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```
Comment (aguibert) [inline pom.xml:58]: Can you remove the dependencyManagement and just have the regular `<dependencies>`. No need for dependency management on a single-module project

File: pom.xml [hunk 2]
```diff
@@ -34,10 +34,28 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jakarta.json.version>2.0.0-RC1</jakarta.json.version>
+        <jakarta.json.version>2.0.0-RC2</jakarta.json.version>
         <jakarta.json.bind.version>2.0.0-RC1</jakarta.json.bind.version>
+        <jakarta.enterprise.cdi-api.version>3.0.0-M2</jakarta.enterprise.cdi-api.version>
```
Comment (aguibert) [inline pom.xml:39]: Can you please undo all of the CDI related changes? We already have that covered in PR #408

Comment (jGauravGupta) [reply pom.xml:39]: Hi @aguibert,

May you please merge the PR #408 first?

Comment (aguibert) [reply pom.xml:39]: Yep, PR #408 is merged now so you can rebase this one

**Pull request merged**